src/test/java/org/folio/rest/api/StorageTestSuite.java

```diff
@@ -88,7 +89,8 @@
   LegacyItemEffectiveLocationMigrationScriptTest.class,
   IterationJobRunnerTest.class,
   AuthorityStorageTest.class,
-  SampleDataTest.class
+  SampleDataTest.class,
+  KafkaTopicsExistsTest.class
```

Comment:
No need to put KafkaTopicsExistsTest into the StorageTestSuite.
mvn automatically runs all org/folio/services tests: https://github.com/folio-org/mod-inventory-storage/blob/v22.0.2/pom.xml#L458

Reply:
ok